nextflow_schema.json

```
@@ -65,16 +65,16 @@
             "properties": {
                 "seed": {
                     "type": "string",
-                    "enum": ["YASS", "NEAR", "MAM8", "RY128"],
-                    "help_text": "LAST creates a database of seed sequences in the _target_ genome, and provides different ways to generate these seeds. The default (`YASS`) searches for long-and-weak similarities that allow for mismatches but not gaps. Among alternatives, there are `NEAR` for short-and-strong (near-identical) similarities with many gaps (insertions and deletions), `MAM8` to find weak similarities with high sensitivity, but low speed and high memory usage, or `RY128` that reduces run time and memory use, by only seeking seeds at ~1/128 of positions in each sequence, which is useful when the purpose of running this pipeline is only to generate whole-genome dotplots, or when sensitivity for tiny fragments may be unnecessary or undesirable. See <https://gitlab.com/mcfrith/last/-/blob/main/doc/last-seeds.rst> for details.",
+                    "enum": ["YASS", "NEAR", "MAM4", "MAM8", "RY4", "RY8", "RY16", "RY32", "RY64", "RY128"],
+                    "help_text": "LAST creates a database of seed sequences in the _target_ genome, and provides different ways to generate these seeds. The default (`YASS`) searches for long-and-weak similarities that allow for mismatches but not gaps. Among alternatives, there are `MAM8` to find weak similarities with high sensitivity, but low speed and high memory usage, or `RY4` to `RY128` that reduce run time and memory use at the expense of sensitivity, by only seeking seeds at ~1/4 to ~1/128 of positions in each sequence. See <https://gitlab.com/mcfrith/last/-/blob/main/doc/last-seeds.rst> for details.",
```

```suggestion
                    "help_text": "LAST creates a database of seed sequences in the _target_ genome, and provides different ways to generate these seeds. The default (`YASS`) searches for long-and-weak similarities that allow for mismatches but not gaps. Among alternatives, there are `MAM8` to find weak similarities with high sensitivity, but low speed and high memory usage, or `RY4` to `RY128` that reduce run time and memory use at the expense of sensitivity, by only seeking seeds at ~1/4 to ~1/128 of positions in each sequence. See [https://gitlab.com/mcfrith/last/-/blob/main/doc/last-seeds.rst](https://gitlab.com/mcfrith/last/-/blob/main/doc/last-seeds.rst) for details.",
```

Use markdown syntax for links to ensure they render correctly on the nf-core website :)